### PR TITLE
Modify lesson format

### LINKED ISSUE #137: Modification of acceptable values for lesson timings
# Potential areas of improvement
1. Shortcuts for lessons, e.g. Mon, Tue etc.
2. Right now if users type Monday  0900-1100 with 2 spaces between day and time is rejected. Should not be the case. (Modify isValidLesson)

### PR DESCRIPTION
**Pull request by potaotototo**
Fixes #137 

# What is this pull request for
Previously, lessons only accepted days that are fully spelt out

# What this pull request does
Allow shortcuts like "mon", "tue" etc.

### CODE REVIEW

File: src/main/java/tuteez/logic/commands/DeleteCommand.java [hunk 1]
```diff
@@ -61,6 +61,15 @@ public CommandResult execute(Model model) throws CommandException {
             personToDelete = getPersonToDeleteByName(model, targetName);
         }
 
+        if (model.getLastViewedPerson().get().isPresent()) {
+            if (personToDelete.equals(model.getLastViewedPerson().get().get())) {
```
Comment (curtischang2510) [inline src/main/java/tuteez/logic/commands/DeleteCommand.java:65]: I believe it is better to not chain so many getters. Maybe you can abstract the method `model.getLastViewedPerson().get().isPresent()` and `model.getLastViewedPerson().get().get())`

File: src/main/java/tuteez/model/person/lesson/Lesson.java [hunk 2]
```diff
@@ -26,6 +27,27 @@ public class Lesson {
     private static final DateTimeFormatter TIME_FORMATTER = DateTimeFormatter.ofPattern("HHmm");
     private static final String VALID_TIME_RANGE_REGEX = "([01]?[0-9]|2[0-3])[0-5][0-9]-([01]?[0-9]|2[0-3])[0-5][0-9]";
     private static final DateTimeFormatter timeFormatter = DateTimeFormatter.ofPattern("HHmm");
+    /**
+     * Maps various string representations of days (e.g., "monday", "mon") to their corresponding Day enum values.
+     * All keys are stored in lowercase.
+     */
+    private static final Map<String, Day> DAY_NAME_MAP = new HashMap<>();
+    static {
+        DAY_NAME_MAP.put("monday", Day.MONDAY);
```
Comment (curtischang2510) [inline src/main/java/tuteez/model/person/lesson/Lesson.java:36]: I believe this should be in the `Day` class. Also lets discuss on a possible alternative implementation for this. But overall good idea!!

**Review by curtischang2510 — COMMENTED** [review]
LGTM

File: src/main/java/tuteez/logic/commands/DeleteCommand.java (hunk 1 above)
Comment (potaotototo) [reply src/main/java/tuteez/logic/commands/DeleteCommand.java:65]: Noted, will do!

**Pull request merged**